main.py

```diff
@@ -14,3 +16,34 @@ def tweet(message: str):
 
     api = tweepy.API(auth)
     api.update_status(message)
+
+
+def get_todays_official_diary():
+    official_diary= []
+
+    url = os.getenv('URL_API_MARIA_QUITERIA')
+    headers = {
+        'Content-Type': 'application/json',
+        'Authorization': os.getenv('TOKEN')
+    }
+    
+    response = requests.get(url, headers=headers)
+    response_json = response.json()
+    
+    date_today = datetime.date.today()
+    for resp in response_json['results']:
+        if resp['date'] == '2021-08-14': #date_today:
```

Comment:
A API vai retornar sempre os últimos diários. Se a coleta falhar ou se não houverem diários na data, vamos publicar edições repetidas. Então é melhor utilizar os filtros `start_date` e `end_date` da API.
Você pode passar os parâmetros como dicionários. Dá uma olhada: https://docs.python-requests.org/en/master/user/quickstart/#passing-parameters-in-urls

Reply:
Entendi como faz a implementação, mas não entendi bem como lidar com as datas pra não acontecer esse caso de postar edições repetidas. Poderia me dar um exemplo?

Reply:
Aqui que entra a issue #2. :) Mas, por enquanto, vamos apenas pegar a data de hoje (`date.today()`) e passar como parâmetro para a API. Aí vamos executar o esse script diariamente no pipeline.